From b987e8221e4eb9c9bae24821534164779f644395 Mon Sep 17 00:00:00 2001
From: Tomas Hozza <thozza@redhat.com>
Date: Wed, 25 Feb 2015 12:29:08 +0100
Subject: [PATCH] Python3 issue: print used as statement in dnssec-coverage.py

When using Python3, the tool would not print extra newlines

Signed-off-by: Tomas Hozza <thozza@redhat.com>
---
 bin/python/dnssec-coverage.py.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bin/python/dnssec-coverage.py.in b/bin/python/dnssec-coverage.py.in
index 0f352c1..58af516 100755
--- a/bin/python/dnssec-coverage.py.in
+++ b/bin/python/dnssec-coverage.py.in
@@ -326,7 +326,7 @@ def vspace():
     if _firstline:
         _firstline = False
     else:
-        print
+        print()
 
 ############################################################################
 # vreset:
-- 
2.1.0

